druid/src/widget/mod.rs

```diff
@@ -70,3 +70,10 @@ pub use widget_ext::WidgetExt;
 
 mod list;
 pub use crate::widget::list::{List, ListIter};
+
+#[cfg(any(feature = "svg", rustdoc))]
+#[doc(cfg(feature = "svg"))]
+mod svg;
+#[cfg(any(feature = "svg", rustdoc))]
+#[doc(cfg(feature = "svg"))]
```

Comment:
What happened to the `cfg_attr` guards?

Reply:
They weren't doing anything that I could detect. When I run it like this I see `This is supported on feature="svg" only.` messages under the type's methods which isn't amazing.

Reply:
Okay I switched back to cfg_attr, can you confirm this looks good?

Reply:
Looks reasonable; can't test personally right now but it fixes CI and we can always tweak it.